content/operate/kubernetes/deployment/quick-start.md

```diff
@@ -114,7 +114,10 @@ that contains cluster specifications.
 
 The following example creates a minimal Redis Enterprise cluster. See the [RedisEnterpriseCluster API reference](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/redis_enterprise_cluster_api.md) for more information on the various options available.
 
-1. Create a file (`my-rec.yaml`) that defines a Redis Enterprise cluster with three nodes:
+{{<note>}}The RedisEnterpriseCluster does not support custom domain names. Domain names for the REC are in the following format: `<rec-name>.<namespace>.svc.cluster.local`.{{</note>}}
```

Comment:
Maybe this note is a better fit to the "Networking" page? Seems a little out of context in the current location...